src/lib/BasicDropdown/BasicDropdown.svelte

```diff
@@ -25,10 +25,10 @@
    * Label for the dropdown (currently hidden)
    * @type {string}
    */
-  export let inline_label;
+  export let inlineLabel;
 
   /**
-   * placeholder for when no option is selected
+   * placeholder for when no option is selected (accepts a null value)
    * @type {string} [placeholder="Select..."]
```

Comment:
I think the type definition should be updated here to `{string | null}` to avoid warnings when a null value is passed down